integration/test/ParseFileTest.js

```diff
@@ -114,4 +114,16 @@ describe('Parse.File', () => {
       assert.equal(e.code, Parse.Error.FILE_DELETE_ERROR);
     }
   });
+
+  it('failing test case for file upload with base64 data', async () => {
+    const file = new Parse.File('parse-server-logo', { base64: 'ParseA==' });
+    await file.save();
+    let data = await file.getData();
+    assert.equal(data, 'ParseA==');
+    file._data = null;
+    await file.save();
+    assert.equal(file._data, null);
+    data = await file.getData();
+    assert.equal(data, 'ParseA==');
+  });
 });
```

Comment:
I don’t think the following test case would be necessary as all the test cases related to file upload were failing. This is just a redundant test case. Let me know your thoughts @mtrezza.

Reply:
Do you mean the other tests were failing as well for Parse Server using the latest alpha version of the Parse JS SDK?

Reply:
Yes, all the existing test cases using file upload were failing with the latest alpha version of the Parse JS SDK. The below screenshot was the message where I had informed you about it previously.
![Screenshot 2022-10-28 at 10 01 54 AM](https://user-images.githubusercontent.com/22429514/198502990-9175d08c-8bb1-4208-bd0a-4292763fb8fb.png)

Reply:
Do you think we should be removing that test case?

Reply:
If a test for the same functionality already exists, you can remove it; if no other test does what your test does, we can leave it.

If we keep the test, I think it needs a rewrite, because you are setting `file._data = null;`, but that is an internal property and I'm not sure there is an expected behavior we can use in the test. You would probably want to download the file into a new object from the URL.

```
const file = new Parse.File('parse-server-logo', { base64: 'ParseA==' });
await file.save();
let data = await file.getData();
assert.equal(data, 'ParseA==');
file._data = null;
await file.save();
assert.equal(file._data, null);
data = await file.getData();
assert.equal(data, 'ParseA==');
```

Reply:
There are other tests with the same functionality. Let me remove it.

Reply:
ok

Reply:
I have removed this test case. I'll go head and mark this conversation as resolved.